lib/index.js

```diff
@@ -9,7 +9,7 @@ const chalk = require('chalk');
 const TransformDotComponentInvocation = require('./plugins/transform-dot-component-invocation');
 
 const WARNING_SEVERITY = 1;
-const ERROR_SEVERITY = 2;
+const ERROR_SEVERITY   = 2;
```

Comment:
Would you mind resetting this? I do aesthetically prefer them to be aligned, but the cost of maintaining is much higher (e.g. following project history ends up being harder)...

Reply:
I do not mind and agree 100%. I was actually wondering if you'd comment on this or not haha. I'll revert the module exports as well.